doc/source/ray-core/scheduling/ray-oom-prevention.rst

```diff
@@ -49,6 +50,7 @@ The memory monitor is controlled by the following environment variables:
   .. note::
 
       Keep the value of ``RAY_task_oom_retries`` low, below 25, to avoid extremely long delays as it is using exponential backoff.
+      The delay is capped at 10 minutes.
```

Comment:
This seems high, given that in k8s the cap is 5 minutes. Ray tasks usually run much faster than k8s pods, so I'd expect this to be smaller--- maybe 1 minute retry at max?